pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -55,7 +55,8 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
 	secret := &v1.Secret{
 		ObjectMeta: v1.ObjectMeta{
-			Name: binding.Name,
+			Name:      binding.Name,
```

Comment:
This is fine for now, but should be the name of the specified secret if that is set.